Scripts/Connector/Sample connector restore script.ps1

```diff
@@ -0,0 +1,40 @@
+# This script restores a connector that has been backed up.
+# To backup a Connector look at "Sample connector backup script.ps1"
+
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
+
+# BEFORE RUNNING THESE COMMANDS PERFORM THE FOLLOWING STEPS:
+# 1) Copy the C:\ConnectorBackup folder (generated by the backup script) to the machine in which you will restore it
+# 2) Install the latest Connector from https://aka.ms/UPConnector (do no register the Connector)
+ 
+# Stop services
+net stop "Print Connector service"
+net stop PrintConnectorUpdaterSvc
+ 
+# Restore local printers
+c:\windows\system32\spool\tools\PrintBrm.exe -R -F C:\ConnectorBackup\printers.brm
+ 
+# Import UP certificates
+New-Item -Path "CERT:\LocalMachine\PrintProxyStore"
+$pwd = ConvertTo-SecureString -String "password" -AsPlainText -Force
+Get-ChildItem "C:\ConnectorBackup\Certs" | Foreach-Object { Import-PfxCertificate -Password $pwd -FilePath $_.FullName -CertStoreLocation Cert:\localmachine\PrintProxyStore -Exportable }
```

Comment:
are we going to restore the permissions? same for the backup script, should we restore them to what they were (even if that connector will no longer be used)?

Reply:
actually, they might be taking a backup just in case they need it later, so we should restore the previous ACLs